### PR TITLE
Documentation for running locally

### PR DESCRIPTION
**Pull request by ewencluley**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,3 +75,17 @@ or if your changes to the linter are on a fork
...
 Your PR checks should now use the modified rules from your branch.
+
+## Running Brandwatch Superlinter Action Locally
+You can pull this repository locally and build a docker image of the Brandwatch Sueprlinter Action with
+```
+docker build -t BrandwatchLtd/super-linter-action .
+```
+You can then run it against a local codebase using 
+```
+docker run -e RUN_LOCAL=true -v /path/to/local/codebase:/tmp/lint BrandwatchLtd/super-linter-action
```
Comment (timatbw) [inline README.md:86]: what does the `/tmp/lint` bit do?

Comment (ewencluley) [reply README.md:86]: `-v` creates a volume and the part after the colon mounts the directory into the docker container at the /tmp/lint directory. It's where superlinter looks for stuff to lint

Comment (timatbw) [reply README.md:86]: Ah OK, I thought maybe it was the path of a file I'm expected to have there. Worth stating here that the person running this *should* fill in the local-codebase bit, but *should not* mess with the /tmp/lint bit (leave that literally)

Comment (ewencluley) [reply README.md:86]: good point

**Review by timatbw — APPROVED** [review]

**Pull request merged**